src/components/TaskList.module.scss

```diff
@@ -0,0 +1,60 @@
+.list--item {
+  display: flex;
+  flex-direction: row;
+  padding: 0px;
+  gap: 15px;
+  align-items: center;
+  font-size: 1.1em;
+  line-height: 1;
+  border-radius: 8px;
+  margin: 0px 0px 5px;
+  flex-grow: 0.8;
+  border: 1px solid black;
+  grid-auto-flow: column;
+  list-style-type: none;
+}
+
+.tasks {
+  margin-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+}
+
+.ul--tasks--list {
+  margin: 0;
+  width: 100%;
+}
+
+.icon--task-button {
+  height: 20px;
+}
+
+.delete--task-button,
+.edit--task-button {
+  width: 85px;
+  font-style: normal;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 3px;
+  text-align: center;
+  flex-basis: 0px;
```

Comment:
Can be removed

Reply:
No, it actually influences on the location of the buttons and the space between the icon of the button and the text on it

Reply:
Does the `flex-basis: 0px` have any effect? It shows the 3 lines above but the comment relates to that rule only.

Reply:
as we spoke, it does reflects on the location since we have a min-height